client/src/pages/Home.js

```diff
@@ -51,15 +51,14 @@ const MainContainer = styled.div`
   }
 `;
 
-const StyledWelcome = styled.h2`
-  font-size: ${theme.typography.heading.three};
+const StyledWelcome = styled.h1`
+
   text-align: left;
   margin: 0 0 7rem 0;
   font-family: ${theme.typography.font.family.display}, sans-serif;
   font-style: normal;
 
   @media only screen and ${mq.phone.wide.max} {
```

Comment:
For the mobile screen let's keep the font styles for the title and p tags like the designs:

current
<img width="423" alt="Screen Shot 2020-05-14 at 7 02 24 PM" src="https://user-images.githubusercontent.com/42228386/82003613-ad0cd380-9615-11ea-9173-f0086c9d54b0.png">

figma
<img width="292" alt="Screen Shot 2020-05-14 at 7 04 37 PM" src="https://user-images.githubusercontent.com/42228386/82003673-c877de80-9615-11ea-903e-9c265fcac940.png">

Reply:
DONE.